lib/data/wrapper.js

```diff
@@ -171,18 +172,27 @@ const data = {
     },
 
     get: (objectGetInfo, response, log, cb) => {
-        // If objectGetInfo.key exists the md-model-version is 2 or greater.
-        // Otherwise, the objectGetInfo is just the key string.
-        const objGetInfo = (implName === 'sproxyd' || implName === 'cdmi') ?
-            objectGetInfo.key : objectGetInfo;
+        const isMdModelVersion2 = typeof(objectGetInfo) === 'string';
+        const isRequiredStringKey = constants.clientsRequireStringKey[implName];
```

Comment:
so beautiful i could cry 😄

Reply:
even better -> `constants.clientsRequireStringKey[implName] || false` :)

Reply:
already falsey if not true and consistent with the usage of other similarly defined constants